docs/en/defined-terms.rst

```diff
@@ -39,10 +37,18 @@ Below are the description of acronyms and definitions which are useful for furth
      - All public and/or private entities, conforming to a technical profile and accredited by the Federation Authority, that provide citizens with an IT Wallet Instance.
    * - Wallet Attestation
      - Verifiable Attestation, issued by the Wallet Provider, that proves the security compliace of the Wallet Instance.
+   * - Wallet Secure Cryptographic Device
+     - Hardware-backed secure environment for creating, storing, and/or managing cryptographic keys and data. Examples include Secure Elements (SE), Trusted Execution Environments (TEEs), and Strongbox.
    * - Credential Status Attestation
      - Verifiable Attestation proving that a related Digital Credential is not revoked.
-   * - Wallet Attestation Service
-     - Device manufacturer service that allows you to certify the authenticity of the mobile app (Wallet Instance).
+   * - Device Integrity Service
+     - A service provided by device manufacturers that verifies the integrity and authenticity of the app instance (Wallet Instance), as well as certifying the secure storage of private keys generated by the device within its dedicated hardware. It's important to note that the terminology used to describe this service varies among manufacturers.
+   * - Wallet Hardware Keys
+     - During the app initialization, the Wallet Instance generates a pair of keys, one public and one private, which remain valid for the entire duration of the Wallet Instance's life. Functioning as a Master Key for the personal device, these Wallet Hardware Keys are confined to the OS domain and are not designed for signing arbitrary payloads. Their primary role is to provide a unique identification for each Wallet Instance.
+   * - Wallet Hardware Key Tag
```

Comment:
WDYT about `Cryptographic Hardware Key Tag`?

Reply:
Great, renamed [here](https://github.com/italia/eudi-wallet-it-docs/pull/233/commits/dae1758cf47951f1cb44f9f8ac24ac9f3b0beb8c)